src/libraries/Common/src/Interop/Unix/System.Native/Interop.IPPacketInformation.cs

```diff
@@ -17,6 +17,7 @@ internal struct IPPacketInformation
         }
 
         [DllImport(Libraries.SystemNative, EntryPoint = "SystemNative_GetControlMessageBufferSize")]
+        [SuppressGCTransition]
         internal static extern int GetControlMessageBufferSize(bool isIPv4, bool isIPv6);
```

Comment:
This function really should be inlinable, but I believe isn't because `bool` isn't blittable. Changing it to `int` would be a win here.